access-modifier-checker/src/test/java/org.kohsuke.accmod.impl/RestrictedElementImplTest.java

```diff
@@ -0,0 +1,32 @@
+package org.kohsuke.accmod.impl;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
```

Comment:
Personally I do not trust mocks and would recommend a [new IT](https://github.com/kohsuke/access-modifier/tree/master/access-modifier-checker/src/it).